### PR TITLE
botd: added hero section static part

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-278
](https://fpjs.myjetbrains.com/youtrack/issue/WEB-278)
mockup -> https://www.figma.com/file/niHrIRowTbqQhDO7W5CPNf/FPJS-WEB-UI-%2B?node-id=1992%3A16936

### CODE REVIEW

File: src/components/botd/HeroSection/HeroSection.module.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.root {
+  overflow: hidden;
```
Comment (tan-90) [inline src/components/botd/HeroSection/HeroSection.module.scss:2]: Do we need to hide overflow?

File: src/components/botd/HeroSection/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import Container from '../../common/Container'
+import Button from '../../common/Button'
+import { URL } from '../../../constants/content'
+import { ReactComponent as GithubIconSvg } from './GithubSVG.svg'
+
+import styles from './HeroSection.module.scss'
+
+export default function HeroSection() {
+  return (
+    <Section className={styles.root}>
```
Comment (tan-90) [inline src/components/botd/HeroSection/HeroSection.tsx:12]: Can't we make the `Container` be at the root and then use two `Section` components inside of it? That way we wouldn't have nested sections.

File: src/components/botd/HeroSection/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                <GithubIconSvg className={styles.icon} />
+                <span>Github</span>
+              </div>
+            </a>
+          </div>
+        </section>
+        <section className={styles.botDSection}>
+          <div className={styles.botD}>
+            <h2 className={styles.title}>Am I a bot?</h2>
+            <h2 className={styles.subTitle}>You are not a bot</h2>
```
Comment (tan-90) [inline src/components/botd/HeroSection/HeroSection.tsx:36]: How can we be sure? 😆

File: src/pages/botd/index.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  pageContext: GeneratedPageContext
+}
+export default function Botd({ pageContext }: AccountSharingProps) {
+  const breadcrumbs = pageContext.breadcrumb.crumbs
+  const { pathname } = useLocation()
+  let siteMetadata = useSiteMetadata()
+  siteMetadata = {
+    ...siteMetadata,
+    title: 'TODO', //TODO
+    description: 'TODO', //TODO
```
Comment (tan-90) [inline src/pages/botd/index.tsx:21]: Let's not forget to change these before the final merge.

**Review by tan-90 — APPROVED** [review]
No major comments, approving in advance but please take a look.

File: src/components/botd/HeroSection/HeroSection.module.scss (hunk 1 above)
Comment (natapg) [reply src/components/botd/HeroSection/HeroSection.module.scss:2]: we don't actually, I will remove it

**Pull request merged**